amp/TEMPLATE/reamp/modules/gpi-reports/common/Utils.jsx

```diff
@@ -95,7 +95,7 @@ class Utils {
     static getCalendarPrefix(settingsWidget, calendars, translatedFY){
         const fiscalYear = translatedFY ? translatedFY : Constants.FY;
         const cal = calendars.filter( calendar => calendar.id == Utils.getCalendarId(settingsWidget) )[0];
-        return cal.isFiscal ? fiscalYear + ' ' : '';
+        return cal['is-fiscal'] ? fiscalYear + ' ' : '';

```

Comment:
Add this string to the constants file, please.

Reply:
Ok

Reply:
Done!